ml_engine/online_prediction/predict_test.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2016 Google Inc. All Rights Reserved. Licensed under the Apache
+# License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License. You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+"""Tests for predict.py ."""
+import base64
+
+import pytest
+
+from predict import census_to_example_bytes, predict_json
+
+
+MODEL = 'census'
```

Comment:
I'm starting to think you want these files in GoogleCloudPlatform/cloudml-samples/census/something